using-mobile-extensions/adobe-campaign-standard/adobe-campaign-standard-event-reference.md

```diff
@@ -0,0 +1,178 @@
+# Adobe Campaign Standard event reference
+
+## Events handled
+
+### Campaign Request Content  <a id="configuration-response-content"></a>
+
+This event is dispatched from Event Hub for the following scenarios:
+1. Event data containing loaded consequences for the latest registered campaign rules needs to be dispatched
```

Comment:
Please change to "Dispatch the event data that contains the loaded consequences for the latest registered Campaign rules."